app/models/concerns/passportable.rb

```diff
@@ -2,7 +2,8 @@ module Passportable
   extend ActiveSupport::Concern
 
   delegate :age_passported?,
-           :benefit_check_passported?, to: :means_passporter
+           :benefit_check_passported?,
+           :not_means_tested?, to: :means_passporter
```

Comment:
not_means_tested was easiest to think of in terms of passporting, and is actually very similar to u18